File: src/URI/URI_1038.java
package URI;
import java.util.Locale;
import java.util.Scanner;

public class URI_1038 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
				
		double preco;		
		int codigo = sc.nextInt();
		int qtde = sc.nextInt();
		
		switch(codigo) {
		case 1:
			preco = qtde * 4.00;
			break;
		case 2:
			preco = qtde * 4.50;
			break;
		case 3:
			preco = qtde * 5.00;
			break;
		case 4:
			preco = qtde * 2.00;
			break;
		case 5:
			preco = qtde * 1.50;
			break;	
		default:
			preco = 0.0;
		}
		
		System.out.printf("Total: R$ %.2f%n", preco);
		
		sc.close();
		

	}

}

File: kotlin/URI_1038.kt
import java.util.*

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)
    val codigo = sc.nextInt()
    val quantidade = sc.nextInt()
    val total = when (codigo) {
        1 -> quantidade * 4.00
        2 -> quantidade * 4.50
        3 -> quantidade * 5.00
        4 -> quantidade * 2.00
        else -> quantidade * 1.50
    }
    println("Total: R$ ${"%.2f".format(total)}")
}
